src/Entities/Blocks/TextBlock.php

```diff
@@ -12,8 +13,33 @@
  * Class TextBlock
  * @package FiveamCode\LaravelNotionApi\Entities\Blocks
  */
-class TextBlock extends Block
+class TextBlock extends Block implements Modifiable
 {
+    protected static function createTextBlock(TextBlock $textBlock, array|string $textContent): TextBlock
+    {
+        if (is_string($textContent)) {
+            $textContent = [$textContent];
+        }
+
+        $text = [];
+        foreach ($textContent as $textItem) {
+            array_push($text, [
```

Comment:
PHP documentation says:

> Note: If you use array_push() to add one element to the array, it's better to use $array[] = because in that way there is no overhead of calling a function.

https://www.php.net/manual/en/function.array-push.php

I think this applies here, right?

Reply:
✅ done